doc/api/process.markdown

```diff
@@ -16,6 +16,12 @@ continue.
 as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'disconnect'
+
+If Node.js was spawned with an IPC channel, this event will be emitted when
+either [`process.disconnect()`][] in child process or
+[`ChildProcess.disconnect()`] in parent process is called.
```

Comment:
I think it should say "disconnect will be emitted when the IPC channel is closed." because that is actually the case, and then link to https://nodejs.org/api/child_process.html#child_process_event_disconnect for further information, so we don't have multiple copies of subtlely wrong/in-disagreement docs.